### ISSUE TITLE
v3.0.4 Property with 'retain (or strong)' attribute must be of object type

### ISSUE THREAD

**Issue by khallmark** [AFNetworking/AFNetworking#3241]
When including AFNetworking 3.0.4 into my personal library as an s.dependency using CocoaPods, I get the following compile error: 

```
AFNetworking/AFURLSessionManager.h:157:1: Property with 'retain (or strong)' attribute must be of object type
AFNetworking/AFURLSessionManager.h:166:1: Property with 'retain (or strong)' attribute must be of object type
```

The issue is similar to #2792, but in a different file.

The fix is the same as in #2930. I am going to fork and open a pull request.

```
#if OS_OBJECT_USE_OBJC
@property (nonatomic, strong, nullable) dispatch_queue_t completionQueue;
#else
@property (nonatomic, assign, nullable) dispatch_queue_t completionQueue;
#endif

/**
 The dispatch group for `completionBlock`. If `NULL` (default), a private dispatch group is used.
 */
#if OS_OBJECT_USE_OBJC
@property (nonatomic, strong, nullable) dispatch_group_t completionGroup;
#else
@property (nonatomic, assign, nullable) dispatch_group_t completionGroup;
#endif
```

**Comment by khallmark** [comment]
Fixed by https://github.com/AFNetworking/AFNetworking/pull/3242

**Comment by khallmark** [comment]
Fixed by merge of #3242

**Comment by kcharwood** [comment]
Direct comments to #3242
